.ci.yaml

```diff
@@ -374,6 +374,25 @@ targets:
       subshard: "1_3"
       tags: >
         ["framework", "hostonly", "shard", "linux"]
+    # Excludes the following from build tests
+    # /docs, AUTHORS, CODE_OF_CONDUCT.md, CODEOWNERS, CONTRIBUTING.md, README.md, TESTOWNERS
+    runIf:
+      - .github/**
+      - .idea/**
+      - .vscode/**
+      - bin/**
+      - dev/**
+      - examples/**
+      - packages/**
+      - .ci.yaml
```

Comment:
Why is `.ci.yaml` excluded? If someone, say, removes critical dependencies from the section above, we want that to run and fail, not silently skip, right?

Reply:
Nevermind, reading comprehension fail.

This seems dangerous though in general. If someone adds a new important file or directory, they would need to know to come update all of these, otherwise we'll silently miss test coverage, and silently not running tests is pretty much the worst failure mode for CI.

Is the resource savings on docs-only changes really worth that risk?

Reply:
I don't disagree with you. This was motivated by https://github.com/flutter/flutter/pull/148999, which was a docs/ only change that then had to wait for all the build tests to run.
The `runIf` property isn't documented in https://github.com/flutter/cocoon/blob/main/CI_YAML.md, I need to look and see where there might be more docs on this.